packages/gas-fee-controller/src/GasFeeController.test.ts

```diff
@@ -1,6 +1,5 @@
 import sinon, { SinonFakeTimers } from 'sinon';
-import { mocked } from 'ts-jest/utils';
-import { ControllerMessenger } from '../ControllerMessenger';
+import { ControllerMessenger } from '@metamask/base-controller';
```

Comment:
Removing `mocked` is not strictly necessary, but will cause issues if we attempt to upgrade Jest (due to a type definition incompatibility with Jest 28 and `ts-jest`). We can get around it for now by typecasting the mocked module to `jest.Mock<ReturnType<F>, Parameters<F>>` (where `F` is the type for a function)`.